
【异常时间】：2024-02-28 08:49:37,922 [6] <>
【异常级别】：ERROR <>
【异常描述】：程序运行发生异常 <br>【异常类型】：NullReferenceException <br>【异常信息】：Object reference not set to an instance of an object. <br>【堆栈调用】：   at OperateHSL.Services.AgvService.AgvControlService.CloseWeldingRoomData(Int32 position, Int32 type) in D:\co_pro\长沙中联\中联_WCS\Libraries\OperateHSL.Services\AgvService\AgvControlService.cs:line 816<br>   at OperateHSL.Web.Controllers.MesDataController.CloseWeldingRoom(CloseWelding model) in D:\co_pro\长沙中联\中联_WCS\Presentation\OperateHSL.Web\Controllers\MesDataController.cs:line 202<br>   at lambda_method(Closure , Object )<br>   at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()<br>   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)<br>   at System.Threading.Tasks.ValueTask`1.get_Result()<br>   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()<br>   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()<br>   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)<br>   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)<br>   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()<br>   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync() <>
